Makefile

```diff
@@ -280,8 +281,11 @@ v8:
 	export PATH="$(NO_BIN_OVERRIDE_PATH)" && \
 		tools/make-v8.sh $(V8_ARCH).$(BUILDTYPE_LOWER) $(V8_BUILD_OPTIONS)
 
+test/common/knownGlobals.json: test/common/parseEslintConfigForKnownGlobals.js lib/.eslintrc.yaml
+	$(NODE) $< > $@
+
 .PHONY: jstest
-jstest: build-addons build-js-native-api-tests build-node-api-tests ## Runs addon tests and JS tests
+jstest: build-addons build-js-native-api-tests build-node-api-tests test/common/knownGlobals.json ## Runs addon tests and JS tests
```

Comment:
I think this is going to break `jstest` (and targets such as `test-only`) for builds from the source tar ball as we strip out anything related to linting (e.g. `lib/.eslintrc.yaml`) from those: https://github.com/nodejs/node/issues/5618
https://github.com/nodejs/node/blob/aa97c9d973b336f0e92aa3a5f60e7b24a6053c6c/Makefile#L1167